app/src/main/res/values/strings.xml

```diff
@@ -171,91 +171,92 @@
+    <string name="data_source_goingelectric_desc">Great in germannic countries. Descriptions in German. Community-maintained.</string>
+    <string name="data_source_openchargemap_desc"><![CDATA[Worldwide, with varying quality. Descriptions in English or the local language. Community-maintained and open government-data in some countries (e.g. North America, UK, France, Norway).]]></string>
     <string name="next">next</string>
-    <string name="get_started">Get started</string>
-    <string name="got_it">Got it</string>
+    <string name="get_started">OK</string>
+    <string name="got_it">OK</string>
     <string name="lets_go">Let\'s go</string>
-    <string name="crash_report_text">Sorry, it seems that EVMap has crashed. Please send a crash report to the developer.</string>
-    <string name="crash_report_comment_prompt">You can add a comment below:</string>
+    <string name="crash_report_text">EVMap crashed. Please send a crash report to the developer.</string>
+    <string name="crash_report_comment_prompt">Comment below:</string>
     <string name="powered_by_mapbox">powered by Mapbox</string>
-    <string name="pref_search_provider">Place search provider</string>
-    <string name="pref_search_provider_info"><![CDATA[Data for place search, especially from Google Maps, is relatively expensive. If you use this feature often, please consider making a donation through \"About EVMap -> Donate\".]]></string>
+    <string name="pref_search_provider">Geo-searches</string>
+    <string name="pref_search_provider_info"><![CDATA[Data for geo-searches. Expensive to fetch, especially from Google Maps. Please consider donating through “About” → “Donate”.]]></string>
     <string name="github_sponsors">GitHub Sponsors</string>
-    <string name="donate_desc">Support EVMap\'s development with a one-time donation</string>
-    <string name="github_sponsors_desc">Support EVMap on GitHub Sponsors</string>
+    <string name="donate_desc">Make a one-time donation to support EVMap development</string>
+    <string name="github_sponsors_desc">Support EVMap</string>
     <string name="unnamed_filter_profile">Unnamed filter profile</string>
     <string name="privacy_link">https://evmap.vonforst.net/en/privacy.html</string>
     <string name="faq_link">https://evmap.vonforst.net/en/faq.html</string>
     <string name="chargeprice_faq_link">https://evmap.vonforst.net/en/chargeprice_faq.html</string>
     <string name="required">required</string>
     <string name="edit_filter_profile">Edit “%s”</string>
-    <string name="pref_search_delete_recent">Delete recent search results</string>
-    <string name="deleted_recent_search_results">Recent search results have been deleted</string>
-    <string name="settings_data_sources">Data sources</string>
+    <string name="pref_search_delete_recent">Delete recent searches</string>
+    <string name="deleted_recent_search_results">Recent searches deleted</string>
+    <string name="settings_data_sources">Sources</string>
```

Comment:
Just "sources" as the title of that settings section might be too unspecific... Could refer to code, water, scientific papers or anything else... 😉

Reply:
Of which there are only one kind in the app?
Translators have the key to figure out what kind it is.